Formula/llvm.rb

```diff
@@ -381,8 +402,21 @@ def caveats
               "-I#{MacOS.sdk_path}/usr/include",
               "-L#{lib}",
               "-Wl,-rpath,#{lib}", "test.cpp", "-o", "test"
-      assert_includes MachO::Tools.dylibs("test"), "#{opt_lib}/libc++.1.dylib"
+      assert_includes MachO::Tools.dylibs("test"), "@rpath/libc++.1.dylib"
```

Comment:
Aye, see the brief discussion above. I thought the change that broke `llvm`'s test was going to get reverted, but I presume it is sticking around at least for now.